src/style.css

```
@@ -1 +1,8 @@
-/* styles go here */
+.input {
+  margin-bottom: 10px;
+  display: inline-block;
+}
```

And you can easly style it like this:
```suggestion
.form-field {
  margin-bottom: 10px;
}
```